apps/rpc/src/modules/mark/personal-mark-service.ts

```diff
@@ -98,25 +98,25 @@ export function getPersonalMarkService(
         const mark = marks.find((m) => m.id === pm.markId)
 
         if (!mark) return acc
-        if (isPast(getExpiryDate(pm.createdAt, mark.duration))) return acc
+        if (isPast(getPunishmentExpiryDate(pm.createdAt, mark.duration))) return acc
 
         return acc + mark.weight
       }, 0)
 
       if (currentMarkWeight >= 6) {
-        return { suspended: true }
+        return { delay: 0, suspended: true }
```

Comment:
I would prefer the delay here to be in milliseconds. There is also no documentation on what unit this is

Reply:
Might as well